d-rita/fast-food-fast-react
# #164353008 Set up testing environment

**What does this PR do?**
This PR sets up a testing environment for the application.

**Description of tasks completed**
- Install jest and enzyme
- Set up jest.config.js and enzyme.config.js

**How should this be manually tested?**
- Fetch and check out this branch using `git fetch origin ch-setup-testing-164353008 && git checkout ch-setup-testing-164353008`
- Install dependencies using `yarn install`
- Run tests using `yarn test`

**Related Pivotal Tracker Story**
[#164353008](https://www.pivotaltracker.com/story/show/164353008)
